components/CardHorizontal.tsx

```diff
@@ -0,0 +1,28 @@
+import { Flex, Heading, Image, Text } from "@chakra-ui/react";
+
+export type OfferProps = {
+  img: string;
+  name: string;
+  description: string;
+};
+
+export const CardHorizontal = ({ img, name, description }: OfferProps) => (
+  <Flex
+    flexDir="column"
+    gap="4"
+    justifyContent="center"
+    alignItems="start"
+    alignSelf="flex-start"
+    textColor="white"
+    color="white"
+  >
+    <Heading 
```

Comment:
Would be nice if the heading was next to the picture on mobile (see mobile version of o nas page in Figma)